### DISCUSSION TITLE
Allow customizing columns in view assigned assets table

### DISCUSSION THREAD

**Discussion by ghost** [snipe/snipe-it#13089] [Q&A] [answered]
Hey,

under /account/view-assets#asset you can see assigned assets to yourself but where and how do I edit which columns are shown?

**Comment by marcusmoore** [comment]
Hi @HablaDabla, clicking this icon let's you update the columns that are displayed

<img width="533" alt="CleanShot 2023-05-25 at 13 04 07@2x" src="https://github.com/snipe/snipe-it/assets/1141514/07499a3d-8db9-4797-b9fa-26b93ae2ad76">

**Reply by ghost** [reply]
Hello @marcusmoore, thanks for the reply and sorry for the confusion but I was implying how would I add more columns that I wish to show? For example the date of purchase.

**Reply by marcusmoore** [reply]
Right now the columns are [hard-coded](https://github.com/snipe/snipe-it/blob/d9a21cce00a096271ac2316e9c532b106ce4824b/resources/views/account/view-assets.blade.php#L399-L417). If you wanted to you could manually add more columns to [the code](https://github.com/snipe/snipe-it/blob/d9a21cce00a096271ac2316e9c532b106ce4824b/resources/views/account/view-assets.blade.php#L423-L458) but that's not ideal for handling future updates.

**Reply by marcusmoore** [reply, marked as answer]
I created an issue for this (#13104). No promises it will be implemented but you can track any progress on it there.
